### DISCUSSION TITLE
No matter which spinner I choose, it doesn't show

### DISCUSSION THREAD

**Discussion by webJose** [spectreconsole/spectre.console#1417] [Q&A] [answered]
Hello.  I am new to Spectre.Console, and I am trying to add a spinner using `AnsiConsole.Status()`.  No matter which known spinner I use, I only see the `/ | \ -` one.

```c#
await AnsiConsole.Status()
    .Spinner(Spinner.Known.Dots8Bit)
    .StartAsync("Searching...", async (context) =>
    {
        references = await PotentialReferencesService.GetPotentialReferencesForObjectNameAsync(settings.ObjectName);
    });
```

Anything missing here?  Thanks.

**Comment by patriksvensson** [comment, marked as answer]
Your terminal does not have UTF-8 output encoding, so Spectre.Console falls back to a compatible spinner (ASCII). You can read more about this here: https://spectreconsole.net/best-practices

**Comment by FrankRay78** [comment]
I've experienced the same as the OP, however before I apply the cmd fix (as explained on best practices), I've found that `Spinner.Known.Star2` perversely works.  `Spinner.Known.Star` doesn't however.

```csharp
        AnsiConsole.Status()
            .Start("Spinning", ctx =>
            {
                ctx.Spinner(Spinner.Known.Star);

                Thread.Sleep(5000);

                ctx.Spinner(Spinner.Known.Star2);

                Thread.Sleep(5000);
            });
```

<img width="304" height="92" alt="image" src="https://github.com/user-attachments/assets/d57c2b51-6871-4f57-ba6e-ce81db211c09" />

<img width="307" height="92" alt="image" src="https://github.com/user-attachments/assets/39acf976-fef1-4d0a-b50e-d217ec2e409f" />
